metadata/health/health_care_facilities.md

```diff
@@ -0,0 +1,83 @@
+# Title
+
+Utah Licensed Health Care Facilities
+
+## ID
+
+a58d176f-5c95-4a45-b8e8-c544aa3b845a
+
+## Brief Summary
+
+Points dataset representing healthcare facilities such as clinics, hospitals, and surgical centers.
+
+## Summary
+
+This dataset contains point locations of health care facilities throughout Utah. Each facility has been licensed by the Division of Licensing and Background Checks (DLBC) under the authority of the [Utah Department of Health and Human Services](https://dhhs.utah.gov/) (DHHS). We developed this layer using [tabular data](https://docs.google.com/spreadsheets/d/e/2PACX-1vT8pLW5yVrV9EY811pE3leFw9u65FtXkMaADaYYdBi0zpjAhrIhaGDQpdQA-jPn8D07DxStjKcWXUcF/pubhtml?gid=1835834361&single=true) from the DHHS.
```

Comment:
Maybe use `This layer was developed using`.... instead

Reply:
We are trying to avoid using passive voice in the metadata and rephrasing it that way leaves the reader not knowing who created the dataset. Did we create it? If not, I can change the "we" to the group or groups that did.